bin/lang-js/src/transpile.ts

```diff
@@ -0,0 +1,77 @@
+import * as _ from "npm:lodash-es";
+import { transpile } from "jsr:@deno/emit";
+import { Debug } from "./debug.ts";
+import * as _worker from "./worker.js";
+
+const debug = Debug("langJs:transpile");
+
+const LOCK_FILE = "/tmp/lang-js-transpile.lock";
```

Comment:
So lang-js will sync now on every transpilation?

Reply:
Yeah, that's the idea I'm going for. This is really only a thing in tests and local dev. In prod we run firecracker, so we won't ever have contention on the lang-js binary